MOBILE_API.xml

```diff
@@ -7311,6 +7312,46 @@
         </param>
     </function>
 
+    <function name="CancelInteraction" functionID="CancelInteractionID" messagetype="request" since="6.0">
```

Comment:
Should probably include a description of the RPC, even though it wasn't included in the proposal

Reply:
added in https://github.com/smartdevicelink/rpc_spec/pull/178/commits/586b711f1f39a93f882edd0993735dd809162f31